src/ResourceManager/KeyVault/Commands.KeyVault/Commands/RemoveAzureKeyVaultCertificate.cs

```diff
@@ -27,7 +27,7 @@ namespace Microsoft.Azure.Commands.KeyVault
         SupportsShouldProcess = true,
         ConfirmImpact = ConfirmImpact.High,
         HelpUri = Constants.KeyVaultHelpUri)]
-    [OutputType(typeof(KeyVaultCertificate))]
+    [OutputType(typeof(DeletedCertificateBundle))]
```

Comment:
@dragav the object being written to the output stream is still a `KeyVaultCertificate` on line 111, so the output type has not changed